typescript/nomad-deploy/config/testnets/kovan.ts

```diff
@@ -28,7 +28,7 @@ export const devConfig: CoreConfig = {
   updater: '0x4177372FD9581ceb2367e0Ce84adC5DAD9DF8D55',
   optimisticSeconds: 10,
   watchers: ['0x20aC2FD664bA5406A7262967C34107e708dCb18E'],
-  recoveryTimelock: 180,
+  recoveryTimelock: 180, // 3 minutes
```

Comment:
is this correct?

Reply:
correct as in, 180 seconds = 3 minutes? or, correct as in, we want the timelock to be 3 minutes in dev?